src/promotion-builder/features/save-promotions/main-layout/SavePromotion.js

```diff
@@ -107,61 +118,63 @@ class SavePromotion extends Component {
 
     if (view === SAVE_PROMOTION) {
       return (
-        <section className="PromotionBuilder">
-          <form className="promotionDetailsForm">
-            <h2 className="NewPromotionTitle">
-              {name ? name : "new promotion"}
-            </h2>
-            <CheckboxGroup
-              value={devices}
-              name="devices"
-              displayItems={DEVICE_LABELS}
-              legendText="Devices:"
-              onFormChange={this.onFormChange}
+        <main className={b("dynamic__content")}>
+          <section className="PromotionBuilder">
+            <form className="promotionDetailsForm">
+              <h2 className="NewPromotionTitle">
+                {name ? name : "new promotion"}
+              </h2>
+              <CheckboxGroup
+                value={devices}
+                name="devices"
+                displayItems={DEVICE_LABELS}
+                legendText="Devices:"
+                onFormChange={this.onFormChange}
+              />
+              <CheckboxGroup
+                value={ventures}
+                name="ventures"
+                displayItems={VENTURE_LABELS}
+                legendText="Ventures:"
+                onFormChange={this.onFormChange}
+              />
+              <InputBox
+                value={url}
+                name="url"
+                onFormChange={this.onFormChange}
+                label={"URL:"}
+              />
+              <InputBox
+                value={name}
+                name="name"
+                onFormChange={this.onFormChange}
+                label={"Name:"}
+              />
+              <input
+                disabled={isSaveFormDataValid(this.state.formData)}
```

Comment:
I would call this isFormDataValid(), or, isDataValid().

Reply:
done